addons/mail/static/src/discuss/call/common/call_button_mobile.js

```diff
@@ -0,0 +1,32 @@
+/* @odoo-module */
+
+import { Component, useEnv, useState } from "@odoo/owl";
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+export class CallButtonMobileView extends Component {
+    static props = [];
+    static template = "discuss.CallButtonMobileView";
+
+    setup() {
+        this.env = useEnv();
+        this.discussCoreCommonService = useState(useService("discuss.core.common"));
+        this.rtc = useState(useService("discuss.rtc"));
+        this.store = useState(useService("mail.store"));
+        this.ui = useState(useService("ui"));
+    }
+
+    async onClickStartMeeting() {
+        const thread = await this.discussCoreCommonService.createGroupChat({
+            default_display_mode: "video_full_screen",
+            partners_to: [this.store.self.id],
+        });
+        this.rtc.toggleCall(thread, { video: true });
+        this.env.bus.trigger("call.button.mobileView/openInvite", { thread });
+    }
+}
+
+registry
+    .category("systray")
+    .add("discuss.CallButtonMobileView", { Component: CallButtonMobileView }, { sequence: 99 });
```

Comment:
"Start a Meeting" is available in desktop in the sidebar of Discuss app.
For me it's not okay to bloat the systray in mobile to show start a meeting at all time:
- this button has no reason to be shown in almost all screens of Odoo, when this is a specific feature of Discuss.
- in mobile the screen is small, so we really shouldn't add more items in systray in mobile

I much prefer we put that somewhere else in the mobile Discuss app / messaging menu. For example the button could be shown in the top part of these screens, next to buttons `New Channel` and `New Message`.